services/user/Registry/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[workspace]
+resolver = "2"
+members = ["service", "query", "plugin"]
+
+[workspace.package]
+version = "0.1.0"
+rust-version = "1.64"
+edition = "2021"
+publish = false
+repository = "https://github.com/gofractally/psibase"
+homepage = "https://psibase.io"
+
+[package]
+edition = "2021"
+name = "registry_package"
+version = "0.1.0"
```

Comment:
I think you need the edition and version keys, but I suggest you use `edition.workspace = true` and `edition.version = true` to avoid duplicating the values

Reply:
I did it and now I'm receiving these warnings (when I run cargo psibase package, test etc.):
```
warning: /home/sohdev/Workspace/fractally/psibase/services/user/Registry/Cargo.toml: unused manifest key: package.edition.version
```

This is how the file is looking like:
```
[workspace]
resolver = "2"
members = ["service", "query", "plugin"]

[workspace.package]
version = "0.1.0"
rust-version = "1.64"
edition = "2021"
publish = false
repository = "https://github.com/gofractally/psibase"
homepage = "https://psibase.io"

[package]
name = "registry_package"
edition.workspace = true
edition.version = true

[package.metadata.psibase]
package-name = "AppRegistry"
description = "Registry for psibase apps"
services = ["registry"]

[profile.release]
codegen-units = 1
opt-level = "s"
debug = false
strip = true
lto = true

[lib]
crate-type = ["rlib"]

[dependencies]
registry = { path = "service" }
r-registry = { path = "query" }
registry-plugin = { path = "plugin" }
```

Is that correct?

Reply:
> `unused manifest key: package.edition.version`

I think you meant `version.workspace`

Reply:
Oops, my brain shut of momentarily as I wrote the comment before @sparkplug0025, I meant `version.workspace`  but wrote `edition.version` haha. I updated the original comment to make it correct